### PR TITLE
Support case-insensitive keywords

### PR DESCRIPTION
**Pull request by juliusv**
Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

**Comment by juliusv** [comment]
@Nexucis This is the best way I've found to support case-insensitive keywords / contextual keywords so far, based on the new external token specialization features which were just released in lezer: https://lezer.codemirror.net/docs/guide/#external-tokens

**Comment by juliusv** [comment]
This will likely require some changes to the packages using this.

**Comment by Nexucis** [comment]
Actually I like it better than the previous version. The usage of the keyword was a bit weird from my point of view.
And I don't think it won't change so many things. I'm just wondering if we still need to export the different list of keyword (in the file promql.terms.js)

File: src/tokens.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import {Bool, Ignoring, On, GroupLeft, GroupRight, Offset, Avg, Bottomk, Count, CountValues, Group, Max, Min, Quantile, Stddev, Stdvar, Sum, Topk, By, Without, And, Or, Unless} from './parser.terms.js';
+import { aggregateOpTerms } from './promql.terms.js';
```
Comment (Nexucis) [inline src/tokens.js]: Are you using this import somewhere in the file ? I can't find where

**Comment by Nexucis** [comment]
Overall awesome work !

File: src/tokens.js (hunk 1 above)
Comment (juliusv) [reply src/tokens.js]: Ah I was initially, but then didn't end up using it. Fixed, thanks.

**Pull request merged**